Cargo.toml

```diff
@@ -297,6 +297,7 @@ cfg-types = { path = "libs/types", default-features = false }
 cfg-utils = { path = "libs/utils", default-features = false }
 cfg-mocks = { path = "libs/mocks", default-features = false }
 cfg-test-utils = { path = "libs/test-utils", default-features = false }
+proofs = { path = "libs/proofs", default-features = false }
```

Comment:
Out of curiosity, where are you using this? I'm not able to see it

Reply:
Not sure how it got into there, removed it.